interface/patient_file/summary/add_edit_issue.php

```diff
@@ -835,11 +835,11 @@ function divclick(cb, divid) {
                         <div class="form-group clearfix" id="button-container">
```

Comment:
On line 737 and 741 on the date picker inputs, remove the width 50% attribute because it doesn't look that good.